src/mixins/itext_behavior.mixin.ts

```diff
@@ -521,7 +434,6 @@ export abstract class ITextBehaviorMixin<
       this.selectionStart !== currentStart ||
       this.selectionEnd !== currentEnd
     ) {
-      this.restartCursorIfNeeded();
```

Comment:
important fix
no reason to do this here, in case of a cursor the animation will be called from mouse up